src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1062,6 +1062,45 @@ public void TestConfigPropertiesAreValid()
             configValidator.ValidateConfigProperties();
         }
 
+        /// <summary>
+        /// This method tests that config file is validated correctly and no exceptions are thrown.
+        /// </summary>
+        [TestMethod("Validates the complete config."), TestCategory(TestCategory.MSSQL)]
+        public async Task TestConfigIsValid()
+        {
+            TestHelper.SetupDatabaseEnvironment(MSSQL_ENVIRONMENT);
+            DataSource dataSource = new(DatabaseType.MSSQL,
```

Comment:
Could we include a test case where the dataSource is invalid in the integration test as well

Reply:
those tests already exists

Reply:
i've also updated the test to use the main integration test config